packages/cli/src/options/paramsOptions.ts

```diff
@@ -32,3 +38,38 @@ export const paramsOptions = ((ObjectKeys(ChainConfig.fields) as unknown) as (ke
   }),
   {}
 );
+
+const terminalArgsToParamsMap: {[K in keyof ITerminalPowArgs]: keyof IChainConfig} = {
+  "terminal-total-difficulty-override": "TERMINAL_TOTAL_DIFFICULTY",
+  "terminal-block-hash-override": "TERMINAL_BLOCK_HASH",
+  "terminal-block-hash-epoch-override": "TERMINAL_BLOCK_HASH_ACTIVATION_EPOCH",
+};
+
+export function parseTerminalPowArgs(args: ITerminalPowArgs): IBeaconParamsUnparsed {
+  const parsedArgs = ObjectKeys(terminalArgsToParamsMap).reduce((beaconParams: Partial<IBeaconParamsUnparsed>, key) => {
+    const paramOption = terminalArgsToParamsMap[key];
+    const value = args[key];
+    if (paramOption != null && value != null) beaconParams[paramOption] = value;
+    return beaconParams;
+  }, {});
+  return parsedArgs;
+}
+
+export const paramsOptions: ICliCommandOptions<IParamsArgs> = {
+  ...paramsOptionsByName,
+
+  "terminal-total-difficulty-override": {
```

Comment:
@g11tech @dapplion do we want to display these options in the help output? If yes, we want `hidden: true` here

Reply:
Ummm I feel these options are for the normal user (in case some attack happens at merge and these options' values are socially communicated ) so left them unhidden. Open for flipping them to hidden